src/App.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+const App = () => <Calculator />;
```

Comment:
- [ ] As pointed out above :point_up: let's kindly make the App component a functional one :-1: